src/test/tools/extents/extents.c

```diff
@@ -72,12 +72,53 @@ main(int argc, char *argv[])
 	if (ret)
 		goto exit_free;
 
-	unsigned e;
-	for (e = 0; e < exts->extents_count; e++)
-		printf("%lu %lu\n",
+	if (argc == 2) {
+		/* print out all extents */
+		for (unsigned e = 0; e < exts->extents_count; e++) {
 			/* extents are in bytes, convert them to sectors */
-			B2SEC(exts->extents[e].offset_physical),
-			B2SEC(exts->extents[e].length));
+			printf("%lu %lu\n",
+				B2SEC(exts->extents[e].offset_physical),
+				B2SEC(exts->extents[e].length));
+		}
+	} else {
+		/* print the physical offset of the given logical one */
+		char *error;
+
+		long unsigned offset = strtoul(argv[2], &error, 10 /* base */);
+		if (*error != '\0') {
```

Comment:
You should also verify that errno is 0 (and set it to 0 before strtoul).

Reply:
Done